test-integration/helpers/task-plan-builder.coffee

```diff
@@ -260,30 +228,60 @@ class ReadingBuilder extends TestHelper
 
     if sections
       # Open the chapter list by clicking the button and waiting for the list to load
-      @openSelectReadingList()
+      if numExercises
```

Comment:
Could this be agnostic to the type of Assignment? That way tests could click "Select Problems" (get to the problem listing screen) without actually having to select problems?